run-gow

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash
+
+readonly SCRIPT_NAME=$(basename "$0")
+readonly SCRIPT_DIR="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+readonly ARGS="$@"
+readonly ARG_COUNT="$#"
+
+function echo_stderr() {
+    local txt="$1"
+    echo "$txt" >&2
+}
+
+function usage() {
+	echo_stderr "Launch the Games on Whales system"
+	echo_stderr
+	echo_stderr "Usage: $SCRIPT_NAME [options]..."
+	echo_stderr
+	echo_stderr "Options:"
+	echo_stderr
+	echo_stderr "  -h, --help"
+	echo_stderr "      This help text."
+	echo_stderr
+	echo_stderr "  --headless"
+	echo_stderr "      If set, run in headless mode. Use this mode if your host does not have an Xorg server running on it."
+	echo_stderr
+	echo_stderr "  --nvidia"
+	echo_stderr "      Use this option if you want to use an Nvidia GPU with Games on Whales"
+	echo_stderr
+	echo_stderr "  -a, --app"
+	echo_stderr "      Specify an application to launch. Can be used multiple times"
+	echo_stderr
+}
+
+launch_env=("env/base.env")
+compose_files=("docker-compose.yml")
+apps=()
+pos_args=()
+
+gpu_type=none
+
+while [ "$#" -gt 0 ]
+do
+	case "$1" in
+	-h|--help)
+		usage
+		exit 0
+		;;
+	--headless)
+		headless="true"
+		;;
+	--nvidia)
+		gpu_type=nvidia
+		;;
+	--intel)
+		gpu_type=intel
+		;;
+	-a|--app)
+        apps+=("$2")
+        shift
+		;;
+	--)
+		break
+		;;
+	-*)
+		echo_stderr "Invalid option '$1'." >&2
+        usage
+		exit 1
+		;;
+	# non-option arguments should also stop processing; everything after will
+	# be passed to docker compose.
+	*)
+		break
+	    ;;
+	esac
+	shift
+done
+
+if [ "$headless" = "true" ]; then
+	launch_env+=("env/headless.env")
+	compose_files+=("compose/headless.yml")
+else
+	launch_env+=("env/host.env")
+fi
+
+if [ -f "env/$gpu_type.env" ]; then
+	launch_env+=("env/$gpu_type.env")
+fi
+if [ -f "compose/$gpu_type.yml" ]; then
+	launch_env+=("compose/$gpu_type.yml")
```

Comment:
this looks like a bug. it should almost certainly be adding to `compose_files` instead of `launch_env`.